hr_employee_cost_history/__manifest__.py

```diff
@@ -14,6 +14,7 @@
     "installable": True,
     "auto_install": False,
     "depends": [
+        "hr_hourly_cost",
         "hr_timesheet",
```

Comment:
You don't need `hr_timesheet` anymore, do you?

Reply:
That was an issue I came across during the migration. 

timesheet_cost was renamed to hourly_cost and shipped in another module that only depended on hr. Thats why I added hr_timesheet so that I could access to account.analytic relational data.